python/tk_desktop/desktop_window.py

```diff
@@ -843,9 +847,7 @@ def slide_finished():
         anim_group.start()
 
     def open_site_in_browser(self):
-        login = ShotgunLogin.get_instance_for_namespace("tk-desktop")
-        connection = login.get_connection()
-        url = connection.base_url
+        url = shotgun.get_sg_connection().base_url
```

Comment:
Yeah, that's a better approach than the one I am taking. I was simply translating the code from the old way of doing things to the new one. But yours is better. And the shotgun.get_associated_sg_base_url() is not new functionality, it's been there for a while: get_associated_sg_config_data is.